mlflow/server/graphql/graphql_schema_extensions.py

```diff
@@ -54,6 +59,16 @@ def resolve_model_versions(self, info):
         parse_dict(input_dict, request_message)
         return mlflow.server.handlers.search_model_versions_impl(request_message).model_versions
 
+    def resolve_metric_history(self, info, metricKey):
```

Comment:
Disclaimer: just thinking out loud 😅 
While it's certainly convenient, I wonder if we'll ever have a use case for fetching sampled metrics history alongside run data (in the same query). After all, all existing scenarios for fetching this data involve lazily loading it for charts, which happens independently. I'm totally fine with introducing nice extras, I just wonder if this change won't add something never to be used but still requiring maintenance (tests, docs etc)

Reply:
Makes sense! I'll remove this for now, I was mainly implementing this to make sure I understood the whole query joining thing haha. I agree that there isn't going to be a use-case for this.